index.html

```diff
@@ -1823,29 +1823,350 @@ <h5><code>invokeaction</code></h5>
-          <code>queryaction</code>, <code>updateaction</code> and
-          <code>cancelaction</code>. These operations do not yet exist in
-          the WoT Thing Description specification (see
+
+        <section id="cancelaction">
+          <h5><code>cancelaction</code></h5>
+          <p>
+            A <code>cancelaction</code> operation is used to cancel an ongoing
+            <code>Action</code> request.
+          </p>
+          <p class="rfc2119-assertion"
+          id="profile-5.2.2.2-thing-protocol-binding-actions-cancelaction-1">
+            A Web Thing which provides <a href="#async-action-response">
+            Asynchronous Action Response</a>s to an <code>invokeaction</code>
+            operation on an <code>Action</code> MAY also support 
+            <code>cancelaction</code> operations on 
+            that same <code>Action</code>.
+            A Web Thing which only provides <a href="#sync-action-response">
+            Synchronous Action Response</a>s to an <code>invokeaction</code> 
+            operation on an <code>Action</code> SHOULD NOT support 
+            <code>cancelaction</code> operations on that same
+            <code>Action</code>.
+          </p>
+          <p class="rfc2119-assertion"
+          id="profile-5.2.2.2-thing-protocol-binding-actions-canncelaction-2">
+            The URL of an <code>ActionStatus</code> resource to be used in a 
+            <code>cancelaction</code> operation MUST be obtained from the 
+            <code>Location</code> header of an <a href="#async-action-response">
+            Asynchronous Action Response</a>.
+          </p>
+          <p class="rfc2119-assertion"
+          id="profile-5.2.2.2-thing-protocol-binding-actions-cancelaction-3">
+            In order to cancel an action request, a Consumer MUST send an HTTP 
+            request to a Web Thing with:
+          </p>
+          <ul>
+            <li>Method set to <code>DELETE</code></li>
+            <li>
+              URL set to the URL of the <code>ActionStatus</code> resource
+            </li>
+          </ul>
+          <pre class="example">
+            DELETE /things/lamp/actions/fade/123e4567-e89b-12d3-a456-426655 HTTP/1.1
+            Host: mythingserver.com
+          </pre>
+          <p class="rfc2119-assertion"
+          id="profile-5.2.2.2-thing-protocol-binding-actions-cancelaction-4">
+            If a Web Thing receives an HTTP request following the format
+            above and the Consumer has permission to cancel the corresponding
+            <code>Action</code> request, then upon successfully cancelling 
+            <code>Action</code> it MUST send an HTTP response with:
+          </p>
+          <ul>
+            <li>Status code set to <code>204</code></li>
+          </ul>
+          <pre class="example">
+            HTTP/1.1 204 No Content
+          </pre>
+        </section>
+
+        <section class="ednote">Another operation under consideration is
```

Comment:
So we may want to consider also "queryallactions" to list all actions (that the consumer has authorization to see).  This is what I meant by the need to support query/List in my earlier comments.  It's useful when a consumer loses the ActionResource link somehow (e.g. reboot, crash, etc).

Reply:
I agree. See #302 and #1200 for discussions on naming.